src/pkg/services/m365/api/graph/errors.go

```diff
@@ -241,6 +243,11 @@ func IsErrItemAlreadyExistsConflict(err error) bool {
 		parseODataErr(err).hasErrorCode(err, nameAlreadyExists)
 }
 
+func IsErrNameAlreadyExistsConflict(err error) bool {
```

Comment:
Can we just use `IsErrItemAlreadyExistsConflict`? Why do we need this new check? Also, I don't see `ErrNameAlreadyExistsConflict` sentinel being used anywhere. Are you planning on using it in a future PR?

Reply:
yes @pandeyabs, will use this for `In-place` restore when there is a `list name` collision

Reply:
Thanks! Why not use existing `IsErrItemAlreadyExistsConflict` function? It's also checking against `nameAlreadyExists` odata error. You can reuse the ` graph.ErrItemAlreadyExistsConflict` sentinel as well like the restore code for other services. Let me know if you disagree.

Reply:
yeah I can use `IsErrItemAlreadyExistsConflict` itself. Closing this PR